src/nvim/testdir/Makefile

```diff
@@ -77,6 +77,7 @@ NEW_TESTS ?= \
 	    test_mksession_utf8.res \
 	    test_nested_function.res \
 	    test_normal.res \
+	    test_number.res \
```

Comment:
should also add `test_findfile.res` in this Makefile, else (I think) running it as a single test won't work (`TEST_FILE=.... make oldtest`).

Reply:
@justinmk done.